From            commit 0f26918f3e3d683e56050cf4d4c9483960bfa5b1 Sat Nov 18 15:42:57 2023
From Author:    osgens <osgens@student.42.fr>
Date:           Sat Nov 18 15:42:57 2023 +0000
Subject:        [PATCH] changed name in Makefile

"changed version name in Makefile"

Signed-off-by: osgens <osgens@student.42.fr>

 ---
diff --git a/Makefile b/Makefile
index 7150ec9..0b5e2c8 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 4
 SUBLEVEL = 12
-EXTRAVERSION = ogenser
+EXTRAVERSION = ogenser-thor_kernel
 NAME = ogenser - Hurr durr I'ma ninja sloth

 # *DOCUMENTATION*
